src/application/services/useMarketplace.ts

```diff
@@ -1,35 +1,43 @@
 import type EditorTool from '@/domain/entities/EditorTool';
-import { type Ref, ref, onMounted } from 'vue';
+import { type Ref, ref, onMounted, watch } from 'vue';
 import { marketplaceService } from '@/domain';
+import type { EditorToolWithUserBinding } from '@/domain/entities/EditorTool';
 
 /**
  * Composable for the application state
  */
 interface UseMarketplaceComposable {
   /**
-   * All editor tools that are used in notes creation
+   * Get list of all tools with user binding
```

Comment:
what is user binding?

Reply:
does user plugin belongs to user

Reply:
then call it "list of plugins that the user has installed"

Reply:
changed comment